lua/core/lualine/init.lua

```diff
@@ -1,46 +1,29 @@
-local M = {}
-M.config = function()
-  lvim.builtin.lualine = {
+local M = {
+  defaults = {
     active = true,
-    style = "lvim",
```

Comment:
Same comment as gitsigns. This will remove the autocomplete "feature".

Reply:
Are you talking about the auto-completion when writing `lvim.` ?

Reply:
Yeah basically, otherwise I had wanted to remove them ages ago.

Reply:
Please read this https://github.com/LunarVim/LunarVim/projects/1#card-67485846

Reply:
Closing for the same reason as below